src/service/performance-impl.js

```diff
@@ -121,10 +121,7 @@ export class Performance {
 
     // If Paint Timing API is not supported, cannot determine first contentful paint
     if (!supportedEntryTypes.includes('paint')) {
-      this.metrics_.rejectSignal(
-        TickLabel.FIRST_CONTENTFUL_PAINT,
-        new Error('First Contentful Paint not supported')
-      );
+      this.metrics_.rejectSignal(TickLabel.FIRST_CONTENTFUL_PAINT);
```

Comment:
I think this would be something like 
```suggestion
      this.metrics_.rejectSignal(
        TickLabel.FIRST_CONTENTFUL_PAINT,
        dev().createExpectedError('First Contentful Paint not supported')
      );
```